crates/cargo-winrt/Cargo.toml

```diff
@@ -17,3 +17,4 @@ zip = { version = "0.5", default-features = false, features = ["deflate"] }
 console = { version = "0.11", default-features = false }
 serde_json = "1.0"
 once_cell = "1.4"
+indicatif = "0.15.0"
```

Comment:
Remove the final `.0` since we don't need to rely on a specific minor version number.

Reply:
Fixed